### PR TITLE
Issue 793 ready to merge

### PR DESCRIPTION
**Pull request by mouseratti**
Work is in progress, do not need to merge at the moment

### CODE REVIEW

**Comment by mouseratti** [comment]
@Stibbons can you review my code?

File: src/guake/prefs.py [hunk 1]
```diff
@@ -726,19 +729,57 @@ def reload_erase_combos(self, btn=None):
...
+            self.client.set_int(KEY('/general/display_n'), dest_screen)
+
+        if dest_screen == ALWAYS_ON_PRIMARY:
+            first_item = combo.get_model().get_iter_first()
+            combo.set_active_iter(first_item)
+        else:
+            seen_first = False  # first item "always on primary" is special
+            for i in combo.get_model():
+                if seen_first:
+                    i_int = int(i[0].split()[0])  # extracts 1 from '1' or from '1 (primary)'
```
Comment (Stibbons) [inline src/guake/prefs.py]: That looks sensible to potential translation, isn't there another way to retried this value ? (index?)

Comment (Stibbons) [inline src/guake/prefs.py]: oh, didn't saw this was a copy paste section. Forget my comment

**Comment by Stibbons** [comment]
can you fix the issue with travis?

**Comment by mouseratti** [comment]
I'm trying to do this. I've attached tests to the build and now it doesn't work. Need some time to fix.

**Comment by mouseratti** [comment]
@Stibbons its ready to merge now

**Comment by Stibbons** [comment]
Will test it :) Thanks a lot !

**Pull request merged**